File: langgraph/result/qwen/3b-eq2js/98.js
function tallyEvenIndexVowels(inputString) {
    let vowelCount = 0;
    let index = 0;
    while (index < inputString.length) {
        if (index % 2 === 0 && "AEIOU".includes(inputString[index])) {
            vowelCount++;
        }
        index++;
    }
    return vowelCount;
}

File: langgraph/data/semantic_equiv/98.py
def tally_even_indexed_vowels(input_string):
    vowel_count = 0
    index = 0
    while index < len(input_string):
        if index % 2 == 0 and input_string[index] in "AEIOU":
            vowel_count += 1
        index += 1
    return vowel_count

# method name: count_upper -> tally_even_indexed_vowels
# parameter name: s -> input_string
# variable name: count -> vowel_count, i -> index
# replaced for loop with while loop and added condition for even index